fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryStreamingAggregationPlan.java

```diff
@@ -70,43 +71,58 @@
  * </UL>
  */
 @API(API.Status.INTERNAL)
-public class RecordQueryStreamingAggregatePlan implements RecordQueryPlanWithChild {
+public class RecordQueryStreamingAggregationPlan implements RecordQueryPlanWithChild {
     private static final ObjectPlanHash BASE_HASH = new ObjectPlanHash("Record-Query-Streaming-Aggregator-Plan");
-    public static final Logger LOGGER = LoggerFactory.getLogger(RecordQueryStreamingAggregatePlan.class);
+    public static final Logger LOGGER = LoggerFactory.getLogger(RecordQueryStreamingAggregationPlan.class);
 
     @Nonnull
     private final Quantifier.Physical inner;
     @Nonnull
-    private final Value resultValue;
+    private final AggregateValue aggregateValue;
```

Comment:
I guess the comment above:
> ... Each concrete {@link AggregateValue} (their order in the list determines their order in the output) will calculate the aggregation function (SUM, MIN, MAX etc.) 

could be adapted since now we have a single `aggregateValue`?